### PR TITLE
Rename branding guidelines to branding

### PR DESCRIPTION
**Pull request by aramshiva**
Fixing #4968. Renames `branding_guidelines.html.erb` to `branding.html.erb` and change paths

### CODE REVIEW

**Comment by garyhtou** [comment]
Just a thought, I kinda like the `/branding` path more than `/brand_guidelines`. We'd keep the changes in this PR, but also we'd rename `brand_guidelines` stuff to `branding` (and have `brand_guidelines` redirect to `branding`).

What are your thoughts?

**Comment by aramshiva** [comment]
Good Idea, should we also rename titles to Branding aswell?

**Comment by aramshiva** [comment]
or should we keep it to "Brand Guidelines"

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 class StaticPagesController < ApplicationController
   skip_after_action :verify_authorized # do not force pundit
-  skip_before_action :signed_in_user, only: [:branding, :brand_guidelines, :faq]
-  skip_before_action :redirect_to_onboarding, only: [:branding, :brand_guidelines, :faq]
+  skip_before_action :signed_in_user, only: [:brand_guidelines, :faq]
+  skip_before_action :redirect_to_onboarding, only: [:brand_guidelines, :faq]
```
Comment (garyhtou) [inline app/controllers/static_pages_controller.rb]:
```suggestion
  skip_before_action :signed_in_user, only: [:branding, :faq]
  skip_before_action :redirect_to_onboarding, only: [:branding, :faq]
```

**Review by garyhtou — COMMENTED** [review]
We can keep the "Brand Guidelines" title.

Here are the changes we'll need to make:
- Move content from `branding_guidelines.html.erb` into ` brand.html.erb`
- Update `routes.rb`
	```ruby
    get "brand_guidelines", to: redirect("branding")
    get "branding", to: "static_pages#branding"
    ```
- Rename the controller action from `brand_guidelines` to `branding`
- Update usages of `brand_guidelines_path` to `branding_path`

**Review by garyhtou — APPROVED** [review]
Great! this looks good to me.

Good to merge after @YodaLightsabr takes a look at it

**Review by YodaLightsabr — APPROVED** [review]
Looks good!

**Pull request merged**